test/check-machines-storage-pools

```diff
@@ -695,6 +695,59 @@ class TestMachinesStoragePools(VirtualMachinesCase):
             remove=True,
         ).execute()
 
+    def testGeneralCheckForPoolsDeletion(self):
```

Comment:
There is already a test for pools deletion. It's called testStoragePoolsDeletion. So introducing this with this name is confusing.

I see this test does not actually perform deletion, it's always canceling or erroring. So I guess I can accept it as a seperate test.